### DISCUSSION TITLE
Label for the Intermediate Scattering Function

### DISCUSSION THREAD

**Discussion by Anders-Markvardsen** [ISISNeutronMuon/MDMC#664] [General]
The intermediate scattering function is currently, within the code, labelled `FQt`, which is shorthand for *F(Q,t)*. This is consistent with the e.g. the nMOLDYN paper (Gerald  R.  Kneller  et  al.  “nMOLDYN:  A  program  package  for  a  neutronscattering oriented analysis of Molecular Dynamics simulations” https://www.sciencedirect.com/science/article/abs/pii/001046559500048K). 

However, if you have worked in the field of crystallography (and with ref to the International Tables of Crystallography) using *F* is confusing because *structure factor* and this label refers to a different quantity (the Fourier transform of the density), and *F(Q,t=0)* in the crystallography community equals the intensity of a Bragg reflection and is denoted *I_Q*. You can't really win unfortunately because of the widespread use of the two different meanings of *structure factor* in two very closely related fields (see also https://en.wikipedia.org/wiki/Structure_factor). 

Before we fix #269 and allow users to refine against intermediate scattering function data we should discuss / agree on what label to use for this quantity long-term, and if different from *F(Q,t)*. 

In the spin-echo community it seems *I(Q,t)* is used e.g. see https://tsapps.nist.gov/publication/get_pdf.cfm?pub_id=922151. 

What do people think?

**Comment by hbordallo** [comment]
Suggestion.
If we are going to use nMoldyn, I think we should use the nomenclature that is defined in the program paper. I have attached it. Heloisa
Rog et al. • Vol. 24, No. 5 • Journal of Computational Chemistry
[JCC_24_p657.pdf](https://github.com/MDMCproject/MDMCv0.2_pilot/files/6911203/JCC_24_p657.pdf)

**Reply by Anders-Markvardsen** [reply]
In that paper 

![image](https://user-images.githubusercontent.com/1135476/127813413-4cb38af3-e1c9-4bc5-b726-abc151114fdc.png)

would you be able to ask Gerald why they are using that symbol rather than a "normal" typed capital F? In particular since for dynamical scattering function they use a "normal" typed S, so why different for the intermediate scattering function? Thanks

**Reply by hbordallo** [reply]
Here what I got from Gerald:

Dear Heloisa 

This is my answer to the question answered by Anders : I use the curly F abs S if the
scattering lengths are involved and normal F and S if not. The reason is that the 
physical units are different - they differ by the units of a cross section (length^2).
Is this clear ? 

All the best 
Gerald 

I think it is clear no, no?
Heloisa

**Reply by Anders-Markvardsen** [reply]
Yes and no. No, in the sense that in https://pubs.acs.org/doi/abs/10.1021/acs.jpcc.1c04322 where Gerald is also an author, the notation `F` is used for `curly F` in paper JCC paper above, hence inconsistent. Yes, in that both are F's in some ways, and maybe that is good enough!

**Reply by hbordallo** [reply]
I saw it also. I will ask him again:-) now I am curious! I was hoping that he would extend the explanation.

**Comment by hbordallo** [comment]
E-mail sent:-)
